alfajores.env

```diff
@@ -85,6 +85,15 @@ EIGENDA_LOCAL_ARCHIVE_BLOBS=${EIGENDA_LOCAL_S3_BUCKET:+0}
 #                                ↓ OPTIONAL ↓                                 #
 ###############################################################################
 
+# MONITORING_ENABLED controls whether Grafana and Prometheus are started
+# Set to "true" if you want to launch them, otherwise keep "false"
+MONITORING_ENABLED=false
+
+# GRAFANA_RESTART_POLICY/PROMETHEUS_RESTART_POLICY set the Docker Compose 'restart' policy for Grafana/Prometheus
+# (e.g. 'no', 'always', 'on-failure', or 'unless-stopped')
+GRAFANA_RESTART_POLICY=no
+PROMETHEUS_RESTART_POLICY=no
```

Comment:
Hey @Kourin1996, this looks like it could cause confusion since people could set restart policies like `allways` that conflict with the choice of `MONITORING_ENABLED`.

Instead I think we can just remove the restart policy configuration and set the restart policy in the compose file to be `on-failure` this is how the historical rpc node is manged.